packages/core/src/Stores/notification-store.js

```diff
@@ -1640,4 +1649,50 @@ export default class NotificationStore extends BaseStore {
             platform: 'Account',
         });
     };
+
+    handleAUDCurrencyRemovalNotification = aud_account_closure_status => {
+        const is_funded_account = aud_account_closure_status.status_codes.includes('funded_account');
+        const is_non_funded_account = aud_account_closure_status.status_codes.includes('non_funded_account');
+
+        if (!is_funded_account && !is_non_funded_account) return;
+
+        const notification_header = is_funded_account
+            ? localize('Withdraw your funds')
+            : localize('Change your currency');
+
+        const time_of_closure = getDateFromTimestamp(aud_account_closure_status.time_of_closure);
+
+        const notification_message = is_funded_account ? (
+            <Localize
+                i18n_default_text="AUD accounts won't be available after {{time_of_closure}}."
+                values={{ time_of_closure }}
+            />
+        ) : (
+            <Localize
+                i18n_default_text="AUD accounts won't be available after {{time_of_closure}}. Choose a new account currency."
```

Comment:
seems we are hardcoding AUD currency in FE  here. can we get this currency value from BE

Reply:
will it be better to use the currency config sent inside auth call here instead of hardcoding strings 🙏